lib/livebook_web/live/home_live.ex

```diff
@@ -136,6 +140,17 @@ defmodule LivebookWeb.HomeLive do
           import_opts={@import_opts} />
       </.modal>
     <% end %>
+
+    <%= if @live_action == :edit_sessions do %>
+      <.modal class="w-full max-w-xl" return_to={Routes.home_path(@socket, :page)}>
+        <.live_component module={LivebookWeb.HomeLive.EditSessionsComponent}
+          id="edit-sessions"
+          action={@action}
+          return_to={Routes.home_path(@socket, :page)}
+          sessions={@sessions}
+          selected_session_ids={@selected_session_ids} />
```

Comment:
It will simplify a lot if you instead passed `selected_sessions={selected_sessions(@sessions, @selected_session_ids}` :cat2: